videos/videos.js

```diff
@@ -22,7 +23,7 @@ router.get("/:id", async (req, res) => {
     const video = await Videos.findById(id);
     res.status(200).json(video);
   } catch (err) {
-    res.status(500).json({ message: "Try again later.", err });
+    res.status(500).json({ message: "Try again later 2", err });
```

Comment:
res.status(500).json({ message: "Try again later. Could not find a video with the id provided", err });

Reply:
Yeah good call, I'll fix that real quick. Was just a fast way to find out why we were hitting an error.